org.eclipse.jdt.core.manipulation/core extension/org/eclipse/jdt/internal/corext/fix/InvertEqualsExpressionFixCore.java

```diff
@@ -0,0 +1,120 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Gayan Perera and others.
```

Comment:
Hi @gayanper You need to use the same Copyright owner as QuickAssistProcessor since you have moved code from there.  You can leave your contributors statement as-is.